fdbcli/fdbcli.actor.cpp

```diff
@@ -609,6 +609,9 @@ std::string getDateInfoString(StatusObjectReader statusObj, std::string key) {
 }
 
 std::string getProcessAddressByServerID(StatusObjectReader processesMap, std::string serverID) {
+	if(serverID == "")
+		return "unknown";
```

Comment:
Is this what gets printed if ratekeeper throttles because it couldn't read the ss list? Does it print this in other circumstances too? If it's just this case, I think saying something like "not applicable" is more explicit.

Reply:
Doing a str.find("") always returns 0 and the above function would return a wrong address for the empty string instead of unknown. This change fixes it.

Reply:
The existing behavior for empty string("") which is explicitly handled prints something like

`Performance limited by server: Unable to fetch storage server list
`

I didn't change the existing behavior